lib/VOMSLibrary.py

```diff
@@ -43,6 +44,30 @@ def compare_dates(self, date0, date1):
 		else:
 			return 1
 
+	def parse_vomses_files(self, vomses_dirs: list = ["/etc/vomses", "~/.glite/vomses"]) -> dict:
+		"""Read the files in /etc/vomses and in ~/.glite/vomses, returning a dictionary with the following items:
+		vo: [hosts]"""
+
+		vomses_files = []
+		for vomses_dir in vomses_dirs:
+			if os.path.exists(os.path.expanduser(vomses_dir)):
+				vomses_files.extend(glob.glob(os.path.expanduser(os.path.join(vomses_dir,"*"))))
+		vomses = {}
+		
+		for vomses_file in vomses_files:
+			try:
+				with open(vomses_file) as f:
+					for line in f:
+						if not line.startswith("#"):
+							vo, host, port = line.split()[:3]
```

Comment:
Looking at the parsing done in the C/C++ library, the VO name is the last entry in the line; the first is the alias, whatever that means. In fact the last entry is the "version", which should be optional and I don't know what is for.

Reply:
Ok, I will improve the parsing.